config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: df155bf0d4a6b976d3353d144d683fe2cc4a813adb6be6677373df15a64e12f4c269ef434ae3c1c3ecaf5232beb500d448ba39b72a9bab543ff1408ef864eec3
```

Comment:
Should this file be git ignored to avoid sharing these keys publicly? Or is that only an issue for the prod key (which I see is already hidden)?